cunumeric/array.py

```diff
@@ -1836,6 +1893,64 @@ def choose(self, choices, out=None, mode="raise"):
         else:
             return out_arr
 
+    def compress(self, condition, axis=None, out=None):
```

Comment:
Use `@add_boilerplate("condition")`. Then you don't need to do `out = convert_to_cunumeric_ndarray(out)`.